content/en/docs/collector/howto-receiver-trace.md

```diff
@@ -1,14 +1,14 @@
 # Building a Traces Receiver
 
- If you are reading this tutorial, you probably already have an idea of the OpenTelemetry concepts behind distributed tracing, but if you don't you can quickly read through it [here](https://opentelemetry.io/docs/concepts/signals/traces/).
+ If you are reading this tutorial, you probably already have an idea of the OpenTelemetry concepts behind distributed tracing, but if you don't you can quickly read through it [here](/content/en/docs/concepts/signals/traces.md).
```

Comment:
The link should be just `/docs/concepts/signals/traces`.

Reply:
yup, just fixed it.